e2-studio-star-rx72n-firmware/libs/rx_harq/src/rx_harq.c

```
@@ -133,7 +164,11 @@ rx_err_t rx_chase_combiner_init(rx_chase_combiner_t* combiner, const uint8_t max
 
   combiner->expected_len = 0;
   combiner->count        = 0;
-  combiner->max_combines = (max_combines > 0) ? max_combines : k_harq_default_combines;
+  if (max_combines > k_harq_zero_combines) {
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)